src/seedu/addressbook/AddressBook.java

```diff
@@ -303,16 +302,43 @@ private static void setupDefaultFileForStorage() {
         storageFilePath = DEFAULT_STORAGE_FILEPATH;
         createFileIfMissing(storageFilePath);
     }
-
+    
```

Comment:
Oh I realised why the line has spaces. Due to the way I configured eclipse. Ok got it will fix it.